style.css

```diff
@@ -0,0 +1,60 @@
+@import url('https://fonts.cdnfonts.com/css/google-sans');
+*{
+    margin: 0;
+    padding: 0;
+    font-family: 'Product Sans', sans-serif;
+}
+body{
+    background-color: #5e0ff6;
+    background-image: url('../img/bg.png');
+}
+#header{
+    margin-top: 40px;
+}
+#header h2{
+    color: #fff;
+    text-align: center;
+}
+#header p{
+    text-align: center;
+    color: #fff;
+}
+#content label{
+    color: #fff;
+}
+#content input{
+    background-color: #6e26f4;
+    border: none;
+    padding: 15px;
+    color: #fff;
+    box-shadow: none;
+    margin-bottom: 15px;
+}
+#content textarea{
+    background-color: #6e26f4;
+    border: none;
+    padding: 15px;
+    color: #fff;
+    box-shadow: none;
+    margin-bottom: 15px;
+}
+input::placeholder {
+    color: #eee !important;
+    opacity: 1;
+}
+input:-ms-input-placeholder {
+    color: #eee !important;
+    opacity: 1;
+}
+
+input::-ms-input-placeholder {
+    color: #eee !important;
+    opacity: 1;
+}
+button{
+    background-color: #A7F60F !important;
+    border: none !important;
+    color: #000 !important;
```

Comment:
please avoid Using ! important in CSS should generally be avoided as a best practice, except in certain specific cases where it is necessary to override styles set by other selectors.

Reply:
> Good work here👍Please kindly always include an explanation, a "why", for each requested change. When reviewers provide clear explanations for requested changes, it helps the author to understand the reasons for the change and how to make the necessary improvements. This can lead to higher-quality code that is easier to maintain and extend over time.☺